build.gradle

```diff
@@ -26,7 +24,7 @@ buildscript {
     ]
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.1'
+        classpath 'com.android.tools.build:gradle:7.0.4'
```

Comment:
Reading https://developer.android.com/studio/releases/gradle-plugin it appears this bump will make the minSdkVersion >30 and require JDK 11 for building, is this correct?

Reply:
minSdkVersion >= 23 at least
JDK 11 is required by all new Android apps now